deployment/entity-service/templates/api-ingress.yaml

```diff
@@ -17,10 +21,19 @@ spec:
   - host: {{ . }}
     http:
       paths:
-      - path: /
+      - path: {{ .Values.api.ingress.path }}
+        pathType: {{ .Values.api.ingress.pathType }}
```

Comment:
Oh this isn't quite correct. This was added to k8s at some point.

In [version 1.16 of the k8s reference docs](https://v1-16.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#httpingresspath-v1beta1-extensions) `pathType` isn't part of `apiVersion: extensions/v1beta1`. It is part of the [1.19 reference docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#httpingresspath-v1beta1-extensions) for ingress `apiVersion: extensions/v1beta1`.

I don't think this project offers any guidance as to the supported version of kubernetes but I think this might fail on the CI cluster.